packages/cli/src/commands/list.ts

```diff
@@ -42,14 +47,26 @@ const help = () => {
 
   ${chalk.dim('Examples:')}
 
-  ${chalk.gray('–')} List all deployments
+  ${chalk.gray('–')} List all deployments for the currently linked project
 
     ${chalk.cyan(`$ ${getPkgName()} ls`)}
 
-  ${chalk.gray('–')} List all deployments for the app ${chalk.dim('`my-app`')}
+  ${chalk.gray(
+    '–'
+  )} List all projects in the team of the currently linked project
+
+    ${chalk.cyan(`$ ${getPkgName()} ls --all`)}
 
+  ${chalk.gray('–')} List all deployments for the app ${chalk.dim(
+    '`my-app`'
+  )} in the team of the currently linked project
```

Comment:
Should we call this a "project" instead of an "app"? How does this sound: "List all deployments for the project `my-app`" ?

Reply:
I think that's a great idea. `project` is used a lot more within Vercel than `app`